website/src/pages/index.astro

```diff
@@ -16,6 +17,9 @@ const docs = await Astro.glob('../../../docs/**/*.md');
 		<title>Proxy Memoize</title>
```

Comment:
@Morteza-Mazrae  move these stuff to the layout (the head section, they all should be in one place)